tests/ArgentPonyWarcraftClassicClient.Integration.Tests/GameDataApi/AuctionHouseApiTests.cs

```diff
@@ -34,29 +34,36 @@ public async Task GetAuctionsAsync_Gets_Auctions_ForBCC()
     [ResilientFact]
     public async Task GetAuctionHousesAsync_Gets_AuctionHouseIndex_ForEra()
     {
-        int realmId = 5121;
-        string @namespace = "dynamic-classic1x-us";
+        int realmId = 5265;
+        string @namespace = "dynamic-classic1x-eu";
+        Region region = Region.Europe;
+        Locale locale = Locale.en_US;
+        string host = "eu.api.blizzard.com";
```

Comment:
Minor comment: Although I generally like the introduction of variables to ensure consistency and assign meaningful names to values, I find them less helpful in unit tests.  For example, the use of variables in the URL makes it a little bit harder for a reader to understand what the final URL would look like.  That said, this is a perfectly good approach, and there is no need to make any changes to this pull request.